config/bundles.php

```diff
@@ -1,6 +1,13 @@
 <?php
 
+use CustomerManagementFrameworkBundle\PimcoreCustomerManagementFrameworkBundle;
+use Pimcore\Bundle\EcommerceFrameworkBundle\PimcoreEcommerceFrameworkBundle;
+use Pimcore\Bundle\EcommerceFrameworkBundle\PimcorePaymentProviderPayPalSmartPaymentButtonBundle;
+
 return [
     //Twig\Extra\TwigExtraBundle\TwigExtraBundle::class => ['all' => true],
     Presta\SitemapBundle\PrestaSitemapBundle::class => ['all' => true],
+    PimcoreEcommerceFrameworkBundle::class => ['all' => true],
+    PimcoreCustomerManagementFrameworkBundle::class => ['all' => true],
+    PimcorePaymentProviderPayPalSmartPaymentButtonBundle::class => ['all' => true],
```

Comment:
Missing bundles: `OutputDataConfigToolkitBundle`, `Web2PrintToolsBundle`, `PimcoreDataHubBundle`, `ObjectMergerBundle`

Reply:
@dvesh3 added the missing ones as well.